src/utils/Bootstrap.cpp

```diff
@@ -124,7 +124,7 @@ int Bootstrap::init(int argc, char** argv)
         std::string features = "";
 
         llvm::TargetOptions opt;
```

Comment:
Looks like opt is leaved without initialization, isn't that a problem?

Reply:
I do not get any warning with it... I am not sure what that is xD What do you think?